temporalio/workflow.py

```diff
@@ -424,6 +425,14 @@ class ParentInfo:
     workflow_id: str
 
 
+@dataclass(frozen=True)
+class UpdateInfo:
+    """Information about a workflow update."""
+
+    id: str
+    """Update ID."""
+
```

Comment:
Might be worth chucking the name in here too for use from things a few layers away from the handler

Reply:
What would you call that field? We have `workflow_type` in `Info`.

Reply:
I think `name` works

Reply:
Added `name`